app/views/admin/pages/_slices.html.erb

```diff
@@ -13,12 +13,16 @@ $(function() {
       metaExtraTemplates: <%= raw @page.meta_extra_templates.to_json %>,
 
       super_user: <%= current_admin.super? %>,
+      locale: '<%= I18n.locale %>',
+
+      loadPagePath: '/admin/pages/{{id}}.json?locale=<%= I18n.locale %>',
+      savePagePath: '/admin/pages/{{id}}.json?locale=<%= I18n.locale %>',
```

Comment:
Does anyone know why we'd do this here vs. [here](https://github.com/withassociates/slices/blob/master/app/assets/javascripts/slices/app/slices.js#L12)?